### PR TITLE
[U] adapt to ruby 3.X.

### PR DESCRIPTION
**Pull request by dvarrui**
Change Gemfile so we could use `bundler install` into ruby 3.X.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '~> 2.0'
+# ruby '~> 2.0'
```
Comment (gilbert) [inline Gemfile:2]: Thanks for the PR! Please delete this line instead of leaving as a comment.

File: test/lib/ruby-prolog/ruby-prolog_test.rb [hunk 2]
```diff
@@ -1,8 +1,7 @@
+#!/usr/bin/env ruby
```
Comment (gilbert) [inline test/lib/ruby-prolog/ruby-prolog_test.rb:1]: What's the purpose of this change?

Comment (dvarrui) [reply test/lib/ruby-prolog/ruby-prolog_test.rb:1]: It's my mistake. I fix it! :-)

Sorry!

**Comment by dvarrui** [comment]
.

**Comment by gilbert** [comment]
Hi! Did you mean to close this PR?

**Comment by dvarrui** [comment]
_Sorry for the disturbances! I have little experience working with git collaboratively._

----
This PR has 2 changes:
1. Delete commented line from Gemfile and
2. Add line into test file 

Change 1 has sense but change 2 it is bad idea (it is my mistake) I mean... I tried to execute this test file and add that line but the right it is your way ... using "rake tests" to run test. 

I don't know if it is possible merge 1 and avoid 2 or it is better to close PR and then create new PR with 1.

**Comment by gilbert** [comment]
It's possible to rewrite your branch to remove 2, but you can also create a new PR with just 1. Either is fine :)
